py/emitglue.c

```diff
@@ -201,7 +201,7 @@ mp_obj_t mp_make_function_from_proto_fun(mp_proto_fun_t proto_fun, const mp_modu
     // make the function, depending on the raw code kind
     mp_obj_t fun;
     switch (rc->kind) {
-        #if MICROPY_EMIT_NATIVE
+        #if MICROPY_PERSISTENT_CODE_LOAD_NATIVE
```

Comment:
I'm pretty sure this needs to be `MICROPY_EMIT_NATIVE || MICROPY_PERSISTENT_CODE_LOAD_NATIVE`.

Reply:
That's been corrected, thanks.